client/src/entrypoints/admin/telepath/widgets.js

```diff
@@ -49,17 +50,25 @@ class Widget {
 
   boundWidgetClass = BoundWidget;
 
-  render(placeholder, name, id, initialState, parentCapabilities) {
+  render(placeholder, name, id, initialState, extraAttributes, parentCapabilities) {
     var html = this.html.replace(/__NAME__/g, name).replace(/__ID__/g, id);
     var idForLabel = this.idPattern.replace(/__ID__/g, id);
     var dom = $(html);
+
+    // Add any extra attributes we received to the HTML of the widget
+    if (typeof extraAttributes === "object") {
+      Object.entries(extraAttributes).forEach(([key, value]) => {
+        dom.attr(key, value)
```

Comment:
This is a jquery method and I know we are trying to get rid of jquery. However, most of this component uses jquery so it is better to be consistent.

Maybe we'll rewrite this to remove jquery in the future...